seahorn/stubs/trusty_handle_api_override.c

```diff
@@ -0,0 +1,89 @@
+#include "handle_table.h"
+#include "nondet.h"
+#include "sea_mem_helper.h"
+#include "seahorn/seahorn.h"
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <sys/types.h>
+
+// trusty reference for definitions only
+#include <trusty_ipc.h> // -> ipc structs
+#include <uapi/err.h>   // NO_ERROR definition
+
+/* Waits for an event to occur on a given handle for specified period of time.
+"[retval]: NO_ERROR if a valid event occurred within a specified timeout
+interval; ERR_TIMED_OUT if a specified timeout elapsed but no event has
+occurred; < 0 for other errors"
+*/
+int wait(handle_t handle, uevent_t *event, uint32_t timeout_msecs) {
+  (void)timeout_msecs;
```

Comment:
why no model for timeout? We are more likely to find bugs on exceptional code paths that are hard to test

Reply:
`ERROR_TIMEOUT` is just another negative error code, so I think it's modeled